agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java

```diff
@@ -572,76 +562,30 @@ private static void addLinks(Map<String, String> properties, List<Link> links) {
     }
 
     // TODO revisit this list and behavior of excluding these attributes
-    private static final Set<String> STANDARD_ATTRIBUTE_PREFIXES = ImmutableSet.of("http", "db", "message", "messaging", "rpc", "enduser", "net", "peer", "exception", "thread", "faas");
-
-    private static void addExtraAttributes(Map<String, String> properties, Map<AttributeKey<?>, Object> attributes) {
-        for (Map.Entry<AttributeKey<?>, Object> entry : attributes.entrySet()) {
-            AttributeKey<?> attributeKey = entry.getKey();
-            String stringKey = attributeKey.getKey();
-            int index = stringKey.indexOf(".");
-            String prefix = index == -1 ? stringKey : stringKey.substring(0, index);
-            if (STANDARD_ATTRIBUTE_PREFIXES.contains(prefix)) {
-                continue;
-            }
-            String value = getStringValue(attributeKey, entry.getValue());
-            if (value != null) {
-                properties.put(attributeKey.getKey(), value);
-            }
-        }
-    }
+    private static final Set<String> STANDARD_ATTRIBUTE_PREFIXES =
+            ImmutableSet.of("http", "db", "message", "messaging", "rpc", "enduser", "net", "peer", "exception", "thread", "faas");
 
-    private static void addExtraAttributes(Map<String, String> properties, Attributes attributes) {
+    private static void setExtraAttributes(Map<String, String> properties, ReadableAttributes attributes) {
         attributes.forEach(new AttributeConsumer() {
             @Override
             public <T> void accept(AttributeKey<T> key, T value) {
+                String stringKey = key.getKey();
+                if (stringKey.startsWith("applicationinsights.internal.")) {
```

Comment:
do you mean negation? !startsWith.

Reply:
it's skipping all internal attributes, we don't want to send them to breeze